docs/Documentation.md

```diff
@@ -252,22 +252,20 @@ A default configuration file would look like
   "IndentSpaceNum":4,
   "PageWidth":120,
   "SemicolonAtEndOfLine":false,
-  "SpaceBeforeUnitArgumentInUppercaseInvocation":false,
-  "SpaceBeforeUnitArgumentInLowercaseInvocation":false,
-  "SpaceBeforeParenthesesInUppercaseInvocation":false,
-  "SpaceBeforeParenthesesInLowercaseInvocation":true ,
-  "SpaceBeforeUnitParameterInUppercaseFunctionDefinition":false,
-  "SpaceBeforeUnitParameterInLowercaseFunctionDefinition":false,
-  "SpaceBeforeParenthesesInUppercaseFunctionDefinition":false,
-  "SpaceBeforeParenthesesInLowercaseFunctionDefinition":true ,
+  "SpaceBeforeParameter":true,
+  "SpaceBeforeLowercaseInvocation":true,
+  "SpaceBeforeUppercaseInvocation":false,
   "SpaceBeforeClassConstructor":false,
+  "SpaceBeforeMember":false,
   "SpaceBeforeColon":false,
   "SpaceAfterComma":true ,
   "SpaceBeforeSemicolon": false,
   "SpaceAfterSemicolon":true ,
   "IndentOnTryWith":false,
   "SpaceAroundDelimiter":true ,
   "MaxIfThenElseShortWidth":40,
+  "MaxRecordWidth":40,
```

Comment:
This list is incomplete, [MaxArrayOrListWidth](https://github.com/fsprojects/fantomas/blob/master/src/Fantomas/FormatConfig.fs#L57) is missing for example.

Reply:
oops, done!